http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -255,6 +255,7 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
             userInfo
           )
       }
+      _ = if (samVisibleRuntimesOpt.isEmpty) authProvider.isUserEnabled(userInfo)
```

Comment:
Similar comment to above to reuse existing `match` expression.

Also noting by best practice shouldn't have an `if` without an `else`. In Scala `if..else` is an expression which returns a value.

Also needs to be `_ <- authProvider.checkUserEnabled(userInfo)` since the method returns an `F[..]`. The `<-` makes it `flatMap` the `F`. `=` is for pure expressions.